server/routes/shared/primaryNavBar/primaryNavBarPresenter.ts

```diff
@@ -0,0 +1,44 @@
+import { PrimaryNavItem } from '../../../utils/viewUtils'
+import config from '../../../config'
+import LoggedInUser from '../../../models/loggedInUser'
+
+export default class PrimaryNavBarPresenter {
+  constructor(private readonly active: string, private readonly loggedInUser: LoggedInUser) {}
```

Comment:
Would be nice to have active as a `type` rather than `string` to make it a bit safer

i.e. 

export type navItem = `Referrals` | `Reporting` | `My cases` | `Find interventions`